{"name": "Не удается залогиниться в систему", "status": "passed", "description": "Попытка логина с неправильно введенными или отсутствующими логином или паролем", "parameters": [{"name": "wrong_creds", "value": "{'login': 'WRONG', 'password': 'opa_password_style'}"}, {"name": "status_code", "value": "404"}], "start": 1739452728541, "stop": 1739452728891, "uuid": "427b09c1-766d-425d-9f84-3af0851d6fd5", "historyId": "58b5cf859533c271fd70b911ed0d2881", "testCaseId": "01ceaaa13868027d2f21fb2961bdf35d", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_wrong_password_or_login_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "GrayCat"}, {"name": "thread", "value": "3508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}